web/src/components/metrics.tsx

```diff
@@ -459,7 +459,7 @@ const QueryKebab: React.FC<{ index: number }> = ({ index }) => {
     dispatch(queryBrowserDuplicateQuery(index));
   }, [dispatch, index]);
 
-  const isSpan = (item) => item.title?.props?.children;
+  const isSpan = (item) => item?.title?.props?.children;
```

Comment:
To fix in a follow up PR, but we should implement this as a [Type Guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards), this way typescript could have caught the issue. Also we need to define the right type on the getSpanText function so it does not depend on external logic to catch errors.